src/electionguardtest/export.py

```diff
@@ -0,0 +1,120 @@
+"""
+Sample generation tool to export data from the election.
+
+Specifically constructed to assist with creating sample data.
+The export here is by no means exhaustive or prescriptive of how one
+may choose to export the data for publishing the election.
+
+Refer to the ElectionGuard spec for any specifics.
```

Comment:
@AddressXception  I'm trying to avoid putting the links in the code when the pyproject.toml already includes links to the `electionguard` repo as do the markdown files. Seems overkill to have it also plastered throughout the code.